test/molecule-role/molecule/secrets/Readme.md

```diff
@@ -0,0 +1,81 @@
+
+Secrets configuring
+
+You need to specify path to the decrypting program by specifying `secret_backend_command`
```

Comment:
You need to specify **the** path **of** the decrypting program

Reply:
fixed